curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-046.md

```diff
@@ -0,0 +1,129 @@
+---
+id: 5f3ef6e0835da6ccc3ee9958
+title: Part 46
+challengeType: 0
+isHidden: true
+---
+
+## Description
+<section id='description'>
+
+Within each of the four `article` elements, add the following desserts and prices using an `article` element with two nested `p` elements inside each. Similar to what you did with the coffees, the first `p` element's text should contain the dessert name and the second `p` element's text should contain the price.
+
+```bash
+Donut 1.50
+Cherry Pie 2.75
+Cheesecake 3.00
+Cinammon Roll
+```
```

Comment:
This is similar to one of the steps Kris and I discussed in the HTML Cat Photo App project.  We could break this down into smaller steps, but at some point, it becomes too repetitive.  I suppose we can have them do it once for the Donut like was done for the coffee flavors and then have them finish out for the three remaining desserts.  If you are ok with that, I can split it up as such.

Reply:
I will leave that up to you. You know better what has already been taught and how much more it could be gone through. I do like your suggestion of doing it once, and then doing it for the other three.